### DISCUSSION TITLE
Katana or any kind of old medieval sword

### DISCUSSION THREAD

**Discussion by JustDorien** [HelixNebulaStudio/RiseOfTheDead#529] [Ideas]
In my opinion we should also get something in the game that actually fit zombie style of games, so like a real sword, which would either be a katana or something similiar to zweihänder which is 2 handed weapon

My idea for the katana would be:
base dmg: 445
category for the weapon: edged
stamina cost: 15
swing speed: 0.8s / 0.9s
range: 13/14 studs or whatever this is called

 My idea for the zweihänder would be
 base dmg: 540
 category for the weapon: blunt
 stamina cost: 28 / 29 
 swing speed: 1.2 / 1.4s
 range: 16+

**Comment by Leroy8888** [comment]
You really want this sword in the game don't you?

**Reply by JustDorien** [reply]
Yes, It's my dream actually lol, I've been waiting for katana or zweinhader for over 4 YEARS smh...
